File: src/main/java/Characters/EnemyState.java
package Characters;

public enum EnemyState  {
	// 縄張りモード（自分の縄張りの四隅へ戻る）
    SCATTER,
    // 追跡モード（仙石さんを追いかける）
    CHASE,
    // フィーバーモード（紫になってランダムに逃げる）
    FEVER,
    // 死亡・巣に戻るモード（目玉だけになって巣に戻る）
    DEAD
}

File: src/main/java/Characters/EnemyState.js
// 敵の状態を固定値として定義（JavaのEnumの代わり）
export const EnemyState = Object.freeze({
    SCATTER: "SCATTER", // 縄張りモード
    CHASE:   "CHASE",   // 追跡モード（★これを追加！）
    FEVER:   "FEVER",   // フィーバーモード
    DEAD:    "DEAD"     // 死亡・巣に戻るモード
});